test/test_go.rb

```diff
@@ -1870,29 +1833,17 @@ def test_ctrl_r_multiline
   end
 
   def test_ctrl_r_abort
-    skip("doesn't restore the original line when search is aborted pre Bash 4") if shell == :bash && /(?<= version )\d+/.match(`#{Shell.bash} --version`).to_s.to_i < 4
+    skip("doesn't restore the original line when search is aborted pre Bash 4") if shell == :bash && `#{Shell.bash} --version`[/(?<= version )\d+/].to_i < 4
     %w[foo ' "].each do |query|
-      retries do
```

Comment:
I had to add this `retries` due to Travis CI failure and I believe it helped.

https://github.com/junegunn/fzf/commit/7c40a424c0bf5a8967816d51ead6a71a334f30bb

Can you explain why you decided that retries are no longer needed here?

Reply:
I've seen those intermittent failures have been resolved in recent versions of the fish shell. I can now run the tests in an indefinite loop without failures, whereas before, I agree, you couldn't.

Reply:
Nice to know! Good riddance then.